app/models/mods_behaviors.rb

```diff
@@ -39,52 +39,52 @@ def to_solr(solr_doc = Hash.new, opts = {})
-    solr_doc['subject_title_sim'] = gather_terms(self.find_by_terms(:title_subject))
-    solr_doc['time_sim'] = gather_terms(self.find_by_terms(:temporal_subject))
+    solr_doc['material_ssim'] = "Digital"
+    solr_doc['topical_subject_ssim'] = gather_terms(self.find_by_terms(:topical_subject))
+    solr_doc['geographic_subject_ssim'] = gather_terms(self.find_by_terms(:geographic_subject))
+    solr_doc['temporal_subject_ssim'] = gather_terms(self.find_by_terms(:temporal_subject))
+    solr_doc['occupation_subject_ssim'] = gather_terms(self.find_by_terms(:occupation_subject))
+    solr_doc['person_subject_ssim'] = gather_terms(self.find_by_terms(:person_subject))
+    solr_doc['corporate_subject_ssim'] = gather_terms(self.find_by_terms(:corporate_subject))
+    solr_doc['family_subject_ssim'] = gather_terms(self.find_by_terms(:family_subject))
+    solr_doc['title_subject_ssim'] = gather_terms(self.find_by_terms(:title_subject))
+    solr_doc['time_ssim'] = gather_terms(self.find_by_terms(:temporal_subject))
 
     # TODO: map PBcore's three-letter language codes to full language names
     # Right now, everything's English.
-    solr_doc['language_sim'] = gather_terms(self.find_by_terms(:language_text))
-    solr_doc['language_code_sim'] = gather_terms(self.find_by_terms(:language_code))
-    solr_doc['physical_description_sim'] = gather_terms(self.find_by_terms(:physical_description))
+    solr_doc['language_ssim'] = gather_terms(self.find_by_terms(:language_text))
+    solr_doc['language_code_ssim'] = gather_terms(self.find_by_terms(:language_code))
+    solr_doc['physical_description_ssim'] = gather_terms(self.find_by_terms(:physical_description))
     solr_doc['related_item_url_sim'] = gather_terms(self.find_by_terms(:related_item_url))
     solr_doc['related_item_label_sim'] = gather_terms(self.find_by_terms(:related_item_label))
-    solr_doc['terms_of_use_si'] = (self.find_by_terms(:terms_of_use) - self.find_by_terms(:rights_statement)).text
+    solr_doc['terms_of_use_ssi'] = (self.find_by_terms(:terms_of_use) - self.find_by_terms(:rights_statement)).text
     solr_doc['rights_statement_ssi'] = self.find_by_terms(:rights_statement).text
     solr_doc['other_identifier_sim'] = gather_terms(self.find_by_terms(:other_identifier))
 
     # Extract 4-digit year for creation date facet in Hydra and pub_date facet in Blacklight
     solr_doc['date_ssi'] = self.find_by_terms(:date_issued).text
     solr_doc['date_created_ssi'] = self.find_by_terms(:date_created).text
     # Put both publication date and creation date into the date facet
-    solr_doc['date_sim'] = gather_years(solr_doc['date_ssi'])
-    solr_doc['date_sim'] += gather_years(solr_doc['date_created_ssi']) if solr_doc['date_created_ssi'].present?
+    solr_doc['date_ssim'] = gather_years(solr_doc['date_ssi'])
+    solr_doc['date_ssim'] += gather_years(solr_doc['date_created_ssi']) if solr_doc['date_created_ssi'].present?
```

Comment:
It looks like we have both `date_ssi` and `date_ssim`.  `date_ssi` is `date_issued` and `date_ssim` is a combination of the EDTF exploded out `date_issued` and `date_created` for powering the facet (e.g.199x becomes 1990,1991...1999) .  I'm thinking that we only really need to store `date_ssi` and not `date_ssim` since it is only used for search.  That being the case I wonder if we should rename `date_ssi` to `date_issued_ssi` to match the other renamings.  What do you think?
```suggestion
    solr_doc['date_sim'] = gather_years(solr_doc['date_ssi'])
    solr_doc['date_sim'] += gather_years(solr_doc['date_created_ssi']) if solr_doc['date_created_ssi'].present?
```

Reply:
Yeah, I think renaming the field to `date_issued_ssi` makes sense. And I'll try out changing `date_ssim` to `date_sim`.